recipes-kernel/linux/files/bcm2712-raspberrypi5-usb.dts

```diff
@@ -0,0 +1,262 @@
+/dts-v1/;
+
+/ {
+	#address-cells = <0x2>;
+	#size-cells = <0x2>;
+
+	gic: gic {
```

Comment:
You don't need this node. It is added automatically by xen tools/xenlib

Reply:
We need it to compile this dts - xen replaces it when loading

Reply:
Hmm, AFAIK, Xen provides `GUEST_PHANDLE_GIC`, which should be used like this:

https://github.com/xen-troops/meta-xt-prod-devel-rcar/blob/master/meta-xt-prod-devel-rcar-driver-domain/recipes-kernel/linux/files/r8a7795-salvator-x-4x2g-domd.dts#L15

https://github.com/xen-troops/meta-xt-prod-devel-rcar/blob/master/meta-xt-prod-devel-rcar-driver-domain/recipes-kernel/linux/files/r8a7795-salvator-x-4x2g-domd.dts#L20

I am surprised that it works without this

Reply:
So, As per my understanding this PHANDLE_GIC can be rolled back and `gic` node can be used for domD with dom0less boot. Right? @lorc @dsemenets @oleksiimoisieiev

Reply:
Yes. I am sorry for the fuss :(

 Didn't expected dom0less to work in **that** way